src/main/java/org/apache/flink/graph/streaming/library/BipartitenessCheck.java

```diff
@@ -57,17 +63,17 @@ public static Candidates edgeToCandidate(long v1, long v2) throws Exception {
 	@SuppressWarnings("serial")
 
 	/**
-	 * Implements EdgesFold Interface, applies a function to a vertex neighborhood
-	 * in the {@link GraphWindowStream#foldNeighbors(Object, EdgesFold)} method.
+	 * Implements the EdgesFold Interface, applies foldEdges function to
+	 * a vertex neighborhood
```

Comment:
This is too general and not very informative. What does updateFunction compute in this algorithm specifically?